reana_server/rest/workflows.py

```diff
@@ -256,14 +256,16 @@ def create_workflow():  # noqa
             return jsonify({'message':
                             'Workflow name cannot be a valid UUIDv4.'}), \
                 400
-
+        workflow_dict = {'specification': reana_spec_file['workflow']['spec'],
+                         'type': reana_spec_file['workflow']['type'],
+                         'name': workflow_name}
+        parameters = None
+        if 'inputs' in reana_spec_file:
```

Comment:
Related with https://github.com/reanahub/reana-workflow-controller/pull/113/files#r20517805 we should add in the OpenAPI spec the [reana.yaml schema](https://github.com/reanahub/reana-client/blob/master/reana_client/schemas/reana_analysis_schema.json). I am opening an issue, otherwise we will end up writing reana.yaml schema in two places.